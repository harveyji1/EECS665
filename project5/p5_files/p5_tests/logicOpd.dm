student: class{
    studentID: int;
};

fn: () int{
    hello: int = 1;
    if(hello and true){
        give hello;
    }
}

main: () void{
    fn();
}