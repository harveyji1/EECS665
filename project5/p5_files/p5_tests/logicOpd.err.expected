FATAL [7,8]-[7,13]: Logical operator applied to non-bool operand
Type Analysis Failed